src/messaging/ReliableMessageMgr.cpp

```diff
@@ -320,7 +320,8 @@ CHIP_ERROR ReliableMessageMgr::SendFromRetransTable(RetransTableEntry * entry)
         if (exchangeMgr)
         {
             // After the first failure notify session manager to refresh device data
-            if (entry->sendCount == 1 && mSessionUpdateDelegate != nullptr)
+            if (entry->sendCount == 1 && mSessionUpdateDelegate != nullptr && entry->ec->GetSessionHandle()->IsSecureSession() &&
+                entry->ec->GetSessionHandle()->AsSecureSession()->IsCASESession())
```

Comment:
Why not also PASE sessions over IP? Why just CASE?

Reply:
Because the information we pass in (the "peer id") can't be used to do commissionable discovery to obtain a new IP for a PASE session.  It can only be used for operational discovery.